.eslintrc.json

```diff
@@ -49,80 +46,15 @@
     "z": true,
     "zeta": true
   },
-  "plugins": ["jasmine", "jsdoc", "no-unsanitized", "prettier", "react"],
-  "parser": "babel-eslint",
-  "parserOptions": {
-    "ecmaVersion": 6,
-    "sourceType": "module",
-    "ecmaFeatures": {
-      "jsx": true
-    }
-  },
+  "plugins": ["jasmine"],
   "rules": {
-    "curly": 2,
-    "dot-notation": 2,
-    "jasmine/prefer-toHaveBeenCalledWith": 0,
+    "id-length": 0,
     "jasmine/no-expect-in-setup-teardown": 0,
     "jasmine/no-spec-dupes": 0,
     "jasmine/no-suite-dupes": 0,
+    "jasmine/prefer-toHaveBeenCalledWith": 0,
     "jsdoc/require-hyphen-before-param-description": 2,
```

Comment:
Maybe this one could go to the main? `jsdoc/require-hyphen-before-param-description`